xlive/Blam/Engine/interface/motion_sensor.cpp

```diff
@@ -0,0 +1,25 @@
+#include "stdafx.h"
+#include "motion_sensor.h"
+
+#include "Util/Hooks/Hook.h"
+
+#define SIZE_MULTIPLIER 1
+
+void radar_patch()
+{
+	WriteValue<byte>(Memory::GetAddress(0x2849C4, 0x0), (byte)D3DBLEND_INVSRCCOLOR);
+}
+
+void motion_sensor_apply_patches()
+{
+	if (Memory::IsDedicatedServer()) { return; }
+
+	radar_patch();
+	motion_sensor_fix_size();
+}
+
+void motion_sensor_fix_size()
+{
+	float og_res = *Memory::GetAddress<float*>(0x3E6A9C);
+	WriteValue<float_t>(Memory::GetAddress(0x3E6A9C), og_res * SIZE_MULTIPLIER);
```

Comment:
checked the code and `hud_scale` takes care of this, why is this needed?

Reply:
This increases the resolution of the pulse so that it it will be the correct size on the screen.
If the resolution is not modified the pulse will be smaller than the radar bitmap when the proper size is used

ex. I've used a radar bitmap that's 4x bigger than before when modifying the hud_scale, the blip is still scaled 4x lower than normal
![image](https://user-images.githubusercontent.com/40469582/235472736-d3d47a90-447c-4a3e-ac0d-5b0af91714b5.png)

scaling the value properly fixes this
![image](https://user-images.githubusercontent.com/40469582/235472919-5f364566-1b50-4c7a-8206-b8f9cca8156b.png)